src/Components/Header.jsx

```diff
@@ -0,0 +1,296 @@
+            transform: "rotate(180deg)",
+            background:
+              mode === LightMode
+                ? "linear-gradient(180deg, rgba(238, 236, 250, 0) 0%, #F9F9F9 51.44%, rgba(238, 236, 250, 0) 100%)"
+                : "linear-gradient(180deg, rgba(44, 44, 46, 0) 0%, rgba(84, 84, 88, 0.65) 51.44%, rgba(44, 44, 46, 0) 100%)",
+          }}
+        ></Box>
+
+        <Box
+          css={{
+            width: "53.5% !important",
+            position: "relative",
+          }}
+        >
+          <Input
+            css={{
+              width: "100% !important",
+              color: "$textPrimary",
+            }}
+            value={searchText}
+            onChange={(e) => {
+              getSearchData(e.target.value);
+              setSearchText(e.target.value);
+            }}
+            rightIcon={<SearchIcon />}
+          />
+          {searchText.length > 0 && (
+            <Flex
+              css={{
+                width: "100%",
+                backgroundColor: "$primary",
+                padding: "20px 10px",
+                position: "absolute",
+                border: "1px solid $border",
+                top: 40,
+                borderRadius: "8px",
+                zIndex: "99",
+              }}
+              flexDirection="column"
+              gap="10px"
+            >
+              {searchText.length && dataSearch.length !== 0 ? (
+                dataSearch.map((item) => (
+                  <Text
+                    css={{
+                      fontWeight: 400,
+                      lineHeight: "15px",
+                      borderBottom: "1px solid $border",
+                      paddingBottom: "5px",
+                      color: "$textPrimary",
+                    }}
+                    key={item}
+                  >
+                    {item?.title}
+                  </Text>
+                ))
+              ) : (
+                <Text
+                  css={{
+                    color: "$textPrimary",
+                  }}
+                >
+                  No data
+                </Text>
+              )}
+            </Flex>
+          )}{" "}
+        </Box>
+      </Flex>
+
+      {/* Header right section */}
+      <Flex alignItems="center" justifyContent="center" gap="20px">
+        <Text
+          css={{
+            color: "$textPrimary",
+            fontWeight: "600",
+            fontSize: "$md",
+            width: "120px",
+          }}
+        >
+          Bessie Cooper
+        </Text>
+
+        <DropdownMenu
+          open={isDropDownOpen}
+          onOpenChange={() => setIsDropDownOpen((prev) => !prev)}
+        >
+          <DropdownMenuTrigger asChild>
+            <Flex
+              justifyContent="center"
+              css={{
+                cursor: "pointer",
+              }}
+              alignItems="center"
+              gap="15px"
+            >
+              <img
+                src={Profile}
+                alt=""
```

Comment:
@9xVibee can you add an alternate name instead of empty string, what if the icon fails to load?